deno.jsonc

```diff
@@ -1,17 +1,16 @@
 {
   "$schema": "https://deno.land/x/deno/cli/schemas/config-file.v1.json",
+  "name": "@slack/protocols",
+  "version": "0.0.2",
+  "exports": "./src/mod.ts",
   "fmt": {
-    "files": {
```

Comment:
The `files` and `options` keys are deprecated, and `deno` would warn about using them, so removed them.